packages/core/test/ObjectField.test.jsx

```diff
@@ -5,8 +5,25 @@ import sinon from 'sinon';
 import { UI_GLOBAL_OPTIONS_KEY } from '@rjsf/utils';
 
 import SchemaField from '../src/components/fields/SchemaField';
+import ObjectField from '../src/components/fields/ObjectField';
 import { createFormComponent, createSandbox, submitForm } from './test_utils';
 
+const ObjectFieldTest = (props) => {
```

Comment:
Thanks for adding these... Can you also test with a custom Widget inside an object?